starting_package/openvino_workbench/docker_config_creator.py

```diff
@@ -28,16 +28,17 @@
 from openvino_workbench.constants import DL_WB_DOCKER_CONFIG_PATH, INTERNAL_PORT
 
 
-def does_dir_exist(path_to_dir: str) -> bool:
-    return os.path.isdir(path_to_dir)
-
-
-def check_for_assets_dir(path_to_dir: str):
-    if not does_dir_exist(path_to_dir):
+def check_assets_dir(path_to_dir: str):
+    if not os.path.isdir(path_to_dir):
         print(f'Provided assets directory: "{path_to_dir}" does not exist.\n'
               'Aborting.')
         sys.exit(1)
 
+    if not os.path.isabs(path_to_dir):
```

Comment:
what if we build the absolute path to mount? `os.path.join(current_path, path_to_dir)`